site/docs/plugins/keyboard.md

```diff
@@ -273,6 +273,17 @@ await ctx.reply(text, {
 });
 ```
 
+### Responding to Clicks
+
+As mentioned earlier, all that keyboards do is sending regular text messages.
+Your bot cannot differentiate between ordinary text messages, and text messages that were sent by clicking a button.
+
+Moreover, buttons will always send exactly the message that's written on them.
+Telegram does not allow you to create buttons that display one text, but send another.
```

Comment:
Maybe it should be highlighted that exactly that is possible with the inline keyboard and that its part of why the inline keyboard is far more powerful.